lua/lspconfig/pses.lua

```diff
@@ -0,0 +1,46 @@
+local configs = require 'lspconfig/configs'
+local util = require 'lspconfig/util'
+
+local server_name = "pses"
+local temp_path = vim.fn.stdpath('cache')
+
+local function make_cmd(bundle_path)
+  if bundle_path ~= nil then
+    local command_fmt = [[%s/PowerShellEditorServices/Start-EditorServices.ps1 -BundledModulesPath %s -LogPath %s/pses.log -SessionDetailsPath %s/pses.session.json -FeatureFlags @() -AdditionalModules @() -HostName nvim -HostProfileId 0 -HostVersion 1.0.0 -Stdio -LogLevel Normal]]
+    local command = command_fmt:format(bundle_path, bundle_path, temp_path, temp_path)
+    return {"pwsh", "-NoLogo", "-NoProfile", "-Command", command}
+  end
+end
+
+configs[server_name] = {
+  default_config = {
+    on_new_config = function(new_config, _)
+      local bundle_path = new_config.bundle_path
+      new_config.cmd = make_cmd(bundle_path)
+    end,
+    filetypes = {"ps1"},
+    root_dir = function(fname)
+      return util.find_git_ancestor(fname) or vim.fn.getcwd()
+    end;
+  };
+  docs = {
+    description = [[
+https://github.com/PowerShell/PowerShellEditorServices
+
+Language server for PowerShell.
+
+The default configuration doesn't set cmd unless `bundle_path` is specified.
```

Comment:
I'd make this a bit more clear, mention how to install the server (if it's simple) and then mention briefly what bundle path is.

Reply:
Done